src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -54,10 +54,7 @@
 		</ResourceDictionary>
 	</UserControl.Resources>
 
-	<Grid
-		x:Name="RootGrid"
-		BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}"
-		BorderThickness="0,0,0,1">
```

Comment:
Is it better to apply the border here, or should we do it in the SideBarView?

Reply:
I think the sidebar is better because we can apply the visual states there.

Reply:
Here, sidebar doesn’t have responsibility on styles of inner nav toolbar.

Reply:
Ah true, like mergins?

Reply:
Right

Reply:
Then in sidebar view is better.